windows/src/desktop/help/context/change-hotkey.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Set hotkey
+redirect: ../start/hotkey_set
```

Comment:
Will help.keyman.com need an update to handle `redirect`?

When I copied the file locally, I get

![image](https://user-images.githubusercontent.com/7358010/110407183-03bc3000-80b6-11eb-9ff7-8d7aeaca6040.png)

Reply:
Fixed in https://github.com/keymanapp/keyman.com/pull/246, https://github.com/keymanapp/help.keyman.com/pull/337 and https://github.com/keymanapp/help.keyman.com/pull/338!